test/AppTest/Action/Account/Meetup/EditMeetupActionTest.php

```diff
@@ -0,0 +1,189 @@
+<?php
+declare(strict_types = 1);
+
+namespace AppTest\Action\Account\Meetup;
+
+use App\Action\Account\Meetup\EditMeetupAction;
+use App\Entity\Location;
+use App\Entity\Meetup;
+use App\Service\Location\FindLocationByUuid;
+use App\Service\Meetup\FindMeetupByUuidInterface;
+use Doctrine\ORM\EntityManagerInterface;
+use Ramsey\Uuid\Uuid;
+use Zend\Diactoros\Response;
+use Zend\Diactoros\ServerRequest;
+use Zend\Expressive\Helper\UrlHelper;
+use Zend\Expressive\Template\TemplateRendererInterface;
+use Zend\Form\FormInterface;
+
+/**
+ * @covers \App\Action\Account\Meetup\EditMeetupAction
+ */
+final class EditMeetupActionTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var FormInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $form;
+
+    /**
+     * @var Meetup|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $meetup;
+
+    /**
+     * @var TemplateRendererInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $renderer;
+
+    /**
+     * @var UrlHelper|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $urlHelper;
+
+    /**
+     * @var EntityManagerInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $entityManager;
+
+    /**
+     * @var FindMeetupByUuidInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $findMeetup;
+
+    /**
+     * @var FindLocationByUuid|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $findLocation;
+
+    /**
+     * @var EditMeetupAction
+     */
+    private $action;
+
+    public function setUp()
+    {
+        $this->meetup = $this->createMock(Meetup::class);
+        $this->meetup->expects(self::any())->method('getId')->willReturn(Uuid::uuid4());
+
+        $this->form = $this->createMock(FormInterface::class);
+        $this->renderer = $this->createMock(TemplateRendererInterface::class);
+        $this->urlHelper = $this->createMock(UrlHelper::class);
+        $this->entityManager = $this->createMock(EntityManagerInterface::class);
+        $this->findMeetup = $this->createMock(FindMeetupByUuidInterface::class);
+        $this->findMeetup->expects(self::once())
+            ->method('__invoke')
+            ->with($this->meetup->getId())
+            ->willReturn($this->meetup);
+        $this->findLocation = $this->createMock(FindLocationByUuid::class);
+
+        $this->action = new EditMeetupAction(
+            $this->renderer,
+            $this->form,
+            $this->entityManager,
+            $this->findMeetup,
+            $this->findLocation,
+            $this->urlHelper
+        );
+    }
+
+    public function testGetRequestRendersTemplate()
```

Comment:
It'll just throw an exception, but the exception is tested elsewhere; not necessary to retest in the action IMO